src/Command/ApiCompare.php

```diff
@@ -132,7 +129,7 @@ public function execute(InputInterface $input, OutputInterface $output) : int
             $outputFormats = $input->getOption('format') ?: [];
             Assert::that($outputFormats)->isArray();
 
-            if (in_array('markdown', $outputFormats, true)) {
+            if (ArrayHelpers::stringArrayContainsString('markdown', $outputFormats)) {
```

Comment:
I want to avoid colliding with various tools that say different things here:

 * infection
 * ea inspections
 * phpstan strict checks

Segregating it out was easy peasy, so I just did it, and we have a single location where `in_array()` still exists.

Yes, this is indeed overblown, but I'd be OK with this particular addition, considering that it really is the only thing needed to make mutation tests work without problems (could've been much worse).

Reply:
Another potential issue: contributors (me included) will likely default to just using `in_array` in future contributions, that may slip through the net :/ it's a non-standard approach, which is "unexpected" and inconsistent with current ... "ways" (for want of a better word)

Reply:
No, it won't fail with a matching error message, obviously.

 > the solution is not what people will expect. 

Agreed, but I don't expect that to be a blocker, as a quick review can deal with most of these errors.

 > the solution is unexpected and unconventional.

Agreed again, as I said, the mutation testing outcome was much better than expected, so I'm quite happy with having this deviation from the standard compared to what it could have been if we had much more mutations. 

 > apart from this shouldn't be a problem and that the finger points back to the tools (phpstan, in this specific case)...

Possible, but I don't think that should become a blocker for just a workaround in here. Assuming that we fix phpstan there, we can really just revert this addition (API is marked `@internal` anyway) and that's the end of the story :-P